src/lib/tooltip/OVERVIEW.md

```diff
@@ -0,0 +1,17 @@
+The Angular Material tooltip provides a text label that is displayed with the user hovers 
+over or longpresses an element. 
+
+The tooltip can be displayed above, below, left, or right of the element. By default the position
+will be below. If the tooltip should switch left/right positions in an RTL layout direction, then
+the positions `before` and `after` should be used instead of `left` and `right`, respectively.
+
+| Position  | Description                                                                           |
+|-----------|---------------------------------------------------------------------------------------|
+| `above`   | Always display above the element                                                      |
+| `below `  | Always display beneath the element                                                    |
+| `left`    | Always display to the left of the element                                             |
+| `right`   | Always display to the right of the element                                            |
+| `before`  | Display to the left in left-to-right layout and to the right in right-to-left layout  |
+| `after`   | Display to the right in left-to-right layout and to the right in right-to-left layout |
+
+<!-- example(tooltip-overview) -->
```

Comment:
Added info on the show/hide and their delays. Not sure what else to add about positioning, do you want examples?